package.json

```
@@ -32,6 +36,8 @@
     "@types/node": "^10.7.1",
     "@types/uuid": "^8.3.2",
     "cz-conventional-changelog": "^2.1.0",
+    "@useparagon/whiskers-cli": "0.0.1-canary.15",
+    "@useparagon/whiskers-core": "0.0.1-canary.15",
```

We have a stable version. 🙌 

```suggestion
    "@useparagon/whiskers-cli": "2024.06.14",
    "@useparagon/whiskers-core": "2024.06.14",
```